pkg/virt-config/virt-config.go

```diff
@@ -89,24 +91,19 @@ const (
 )
 
 func IsAMD64(arch string) bool {
-	if arch == "amd64" {
-		return true
-	}
-	return false
+	return arch == "amd64"
```

Comment:
This kind of refactoring belongs to a separate commit IMO, but it's not too big a deal.

Reply:
I can move it in a separate commit bevor adding my s390x code. That should not be a problem when rebasing.

Reply:
I have moved them into a separate commit